src/tool/hpcrun/unwind/common/backtrace.c

```diff
@@ -170,6 +171,8 @@ hpcrun_skip_chords(frame_t* bt_outer, frame_t* bt_inner,
   return &bt_inner[skip];
 }
 
+static _Thread_local int max_unwind_attempts = -1;
```

Comment:
Why is this thread local rather than just a local variable in the backtrace procedure?